fspiop-api/documents/API Definition v2.0-alpha.md

```diff
@@ -3879,17 +4366,20 @@ This section describes complex types used by the API.
 | **payee** | 1 | Party | Information about the Payee in the proposed financial transaction. |
 | **payer** | 1 | Party | Information about the Payer in the proposed financial transaction. |
 | **amount** | 1 | Money | Transaction amount to be sent. |
+| **payeeReceiveAmount** | 0..1 | Money | The amount that the beneficiary will receive |
 | **transactionType** | 1 | TransactionType | Type of the transaction. |
+|**converter** | 0..1 | CurrencyConverter | **PAYER** if the payer DFSP intends to perform currency conversion; **PAYEE** if the payer DFSP wants the payee DFSP to perform currency conversion. If absent and the transfer requires currency conversion, then the converting institution will be inferred from the currency in which the quotation is requested: if the quotation is requested in the source currency, then the payee DFSP should perform currency conversion. If it is in the target currency, then the payer DFSP will perform currency conversion. | Used by the debtor party if it wants to share information about the currency conversion it proposes to make; or if it is required by scheme rules to share this information. This object contains the amount of the transfer in the source and target currencies, but does not identify the FXP being used. |
```

Comment:
"| Used by the debtor party if it wants to share information about the currency conversion it proposes to make; or if it is required by scheme rules to share this information. This object contains the amount of the transfer in the source and target currencies, but does not identify the FXP being used. |"
Seems like a copy-paste error here, same is in line below.